lib/server/utilities/countedDataToHttpView.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef CountedData
+ * @property {Number|Number[]} count
+ * @property {Object[]} rows
+ */
+
+/**
+ * @typedef CountedDataView
```

Comment:
`CountedDataHttpView` (Or I would rather name it `CountedItemsHttpView` or `CountedRowsHttpView`)

Reply:
renmaed to CountedItemsHttpView